### PR TITLE
Fixes the c3.X instances to have the proper disk size

### PR DESCRIPTION
**Pull request by Itxaka**
See subject

### CODE REVIEW

File: libcloud/compute/drivers/ec2.py [hunk 1]
```diff
@@ -219,35 +219,35 @@
         'id': 'c3.large',
         'name': 'Compute Optimized Large Instance',
         'ram': 3750,
-        'disk': 16,
+        'disk': 32,  # x2
```
Comment (sebgoa) [inline libcloud/compute/drivers/ec2.py:222]: I think you can remove all of the # x2 comments

Comment (Itxaka) [inline libcloud/compute/drivers/ec2.py:222]: Its just so its easier to know that its actually 2 different disks. There is a couple of instances in the file which have the same comment, probably so people dont get confused like I was at first :D

Comment (sebgoa) [inline libcloud/compute/drivers/ec2.py:222]: so it's 2 disks of 16 that make 32, or two disks of 32 ?

Comment (Itxaka) [inline libcloud/compute/drivers/ec2.py:222]: 2 total disks, same as it was here https://github.com/Itxaka/libcloud/blob/fix_aws_c3_instances_disks/libcloud/compute/drivers/ec2.py#L336

**Comment by Itxaka** [comment]
@kami ?

**Pull request closed without merging**